guide/latest/cockpit-file.html

```diff
@@ -84,6 +85,12 @@
 <p>It is not an error when the file does not exist. In this case, the
       <code class="code">done()</code> callback will be called with a <code class="code">null</code>
       value for <code class="code">content</code> and <code class="code">tag</code> is <code class="code">"-"</code>.</p>
+<p>Set <code class="code">superuser</code> to "require" for reading or writing a file as root instead of the logged 
```

Comment:
I think we should just say that `superuser` is also allowed and link to the docs for `cockpit.channel`.